app/models/unit.rb

```diff
@@ -45,8 +46,7 @@ def self.permissions
       :provide_feedback,
       :change_project_enrolment,
       :download_stats,
-      :download_grades,
-      :rollover_unit
+      :download_grades
```

Comment:
Please make sure to merge in changes that have already been committed to the repository.